assets/icons/files/tikui.svg

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="16" height="16" version="1.1" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg" xmlns:cc="http://creativecommons.org/ns#" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+ <title>939F69E9-C193-4037-81BF-58C624AF804B</title>
+ <g id="Tikui" transform="translate(-.55489 -.54668)" fill="none" fill-rule="evenodd">
+  <g id="Home" transform="translate(-104,-17)" fill-rule="nonzero">
+   <g transform="translate(104.5,17.5)">
+    <g id="Group" transform="matrix(.24318 0 0 .24318 2.8871 .035331)">
+     <path id="Shape" d="m19.108 49.447c-10.506 0-19.053-8.5237-19.053-19s8.5476-19 19.053-19c10.506 0 19.053 8.5237 19.053 19s-8.5476 19-19.053 19zm0-28.499c-5.2529 0-9.5259 4.261-9.5259 9.4992 0 5.2381 4.273 9.4992 9.5259 9.4992s9.5259-4.261 9.5259-9.4992c0-5.2381-4.273-9.4992-9.5259-9.4992z" fill="#310ca3"/>
+     <ellipse id="Oval" cx="36.255" cy="6.2217" rx="6.1924" ry="6.175" fill="#ffab82"/>
+     <ellipse cx="24.159" cy="59.667" rx="6.1924" ry="6.175" fill="#fa9fb8"/>
+    </g>
```

Comment:
Icons should have only one color, so please keep only the #310ca3, put it in the topmost `g` and add the `data-iconColor="Tikui"` property so that it's color can be changed